src/items.rs

```diff
@@ -1358,9 +1358,10 @@ fn rewrite_fn_base(context: &RewriteContext,
             FnArgLayoutStyle::BlockAlways => false,
             _ => {
                 // If we've already gone multi-line, or the return type would push
-                // over the max width, then put the return type on a new line.
+                // over the max width (including the whitespace after result),
```

Comment:
So, maybe there is some confusion over the word `result` - could you clarify that in the comment please? I assumed it meant `return type`, in which case the space if before since the return type is not pushed on to the result until 1395/1397.